packages/excalidraw/tests/resize.test.tsx

```diff
@@ -895,7 +895,7 @@ describe("multiple selection", () => {
     expect(rightBoundArrow.height).toBeCloseTo(0);
     expect(rightBoundArrow.angle).toEqual(0);
     expect(rightBoundArrow.startBinding).toBeNull();
-    expect(rightBoundArrow.endBinding?.gap).toBeCloseTo(8.0952);
+    expect(rightBoundArrow.endBinding?.gap).toBeCloseTo(7.0952);
```

Comment:
@mtolmacs  investigate if this change is indeed happening because of more precise measurement.